### ISSUE TITLE
BUG-002 - Unable to tab through page successfully

### ISSUE THREAD

**Issue by JosephBanham** [lunarcroissant/react-challenge-1-window-news#2]
This is an Accessibility issue. People who cannot rely on mouse inputs instead rely on the tab key's inputs to navigate through the page - Please test again and fix.

**Comment by lunarcroissant** [comment]
Fixed :)
